src/main/java/com/google/collegeplanner/servlets/PlannerServlet.java

```diff
@@ -29,35 +36,226 @@
 /** Servlet that returns the mutli-semester plan for the given courses.*/
 @WebServlet("/api/planner")
 public class PlannerServlet extends HttpServlet {
+  private HashMap<String, Integer> indegree; // Number of prerequisites for a course
```

Comment:
i think you need to remove a Ramya's commit from your PR so this is clean to review+merge.